docs/content.zh/docs/deployment/filesystems/gcs.md

```diff
@@ -44,7 +44,10 @@ env.readTextFile("gs://<bucket>/<endpoint>");
 stream.writeAsText("gs://<bucket>/<endpoint>");
 
 // Use GCS as checkpoint storage
```

Comment:
```suggestion
// 将 GCS 用作 checkpoint storage
```

Reply:
Thank you for your suggestion. The comment you mentioned is part of a code snippet, and I would rather keep it in English instead of translating it into Chinese, as I believe it will not cause any inconvenience to users.

Reply:
Thanks for the clarification. If we take a look at other case like azure at line 66, oss at line 51, s3 at line 49, they are all translated into Chinese. It is a question of consistency. WDYT?